docs/installation/migrating-to-systemd.md

```diff
@@ -98,3 +38,20 @@ Finally, start the `netbox` and `netbox-rq` services and enable them to initiate
 # systemctl enable netbox.service
 # systemctl enable netbox-rq.service
 ```
+
+You can use the command `systemctl status netbox` to verify that the WSGI service is running:
+
+```
+# systemctl status netbox.service
+● netbox.service - NetBox WSGI Service
+   Loaded: loaded (/etc/systemd/system/netbox.service; enabled; vendor preset: enabled)
+   Active: active (running) since Thu 2019-12-12 19:23:40 UTC; 25s ago
+     Docs: https://netbox.readthedocs.io/en/stable/
+ Main PID: 11993 (gunicorn)
+    Tasks: 6 (limit: 2362)
+   CGroup: /system.slice/netbox.service
+           ├─11993 /usr/bin/python3 /usr/local/bin/gunicorn --pid /var/tmp/netbox.pid --pythonpath /opt/netbox/...
+           ├─12015 /usr/bin/python3 /usr/local/bin/gunicorn --pid /var/tmp/netbox.pid --pythonpath /opt/netbox/...
+           ├─12016 /usr/bin/python3 /usr/local/bin/gunicorn --pid /var/tmp/netbox.pid --pythonpath /opt/netbox/...
+...
+```
```

Comment:
I don't think we need to include this.  Someone who is relatively proficient with systemd will know how to do this and someone who is not won't really know what is going on (beyond the running) and there are other ways to verify that.